src/cache/sqlite.ts

```diff
@@ -141,8 +141,8 @@ class SQLiteAdapter {
   }
 
   removeById(collection, id) {
-    const query = ['DELETE FROM #{collection} WHERE key = ?', [id]];
-    return this.openTransaction(collection, query, null, true)
+    const query = 'DELETE FROM #{collection} WHERE key = ?';
```

Comment:
@georgiwe this makes sense as a straightforward way to express the query. However, isn't the end outcome identical? (refer to [this](https://github.com/Kinvey/nativescript-sdk/blob/0ea4a684e5b2f24906e3085e687a53c765a75409/src/cache/sqlite.ts#L27) ).

The only difference seems to be parameters as `id` vs `[id]`. Was that the problem?